rules/helm/def.bzl

```diff
@@ -131,3 +114,44 @@ template = rule(
         ),
     },
 )
+
+def _version_impl(ctx):
+    script = ctx.actions.declare_file("{}.rb".format(ctx.attr.name))
+    output = ctx.actions.declare_file("version.txt")
+    outputs = [output]
+    contents = """
+        open('{output}', 'w') do |f|
+          f << `{helm} inspect chart {chart}`[/version: (.*)/, 1]
```

Comment:
Would this bubble up unexpected errors correctly?

Reply:
Bazel itself should fail.

Reply:
Actually, we need to add `raise if not $?.success?` to actually handle the exit code.